caom2utils/caom2utils/fits2caom2.py

```diff
@@ -3174,35 +3425,46 @@ def _augment(obs, product_id, uri, blueprint, subject, dumpconfig=False,
                      product_type=ProductType.SCIENCE,
                      release_type=ReleaseType.DATA))
 
+    meta_uri = uri
+    visit_local = None
     if local:
-        if '.header' in local and '.txt' not in local:
-            logging.debug('Using a FitsParser for local file {}'.format(local))
-            parser = FitsParser(get_cadc_headers('file://{}'.format(local)),
-                                blueprint, uri=uri)
-        elif local.endswith('.fits') or local.endswith('.fits.gz'):
-            logging.debug('Using a FitsParser for {}'.format(local))
-            parser = FitsParser(local, blueprint, uri=uri)
+        if uri.startswith('vos'):
```

Comment:
How can local be vos:?

Reply:
'local' is the file on disk. 'uri' is the file in storage as it shows up in the artifact.uri attribute (ad, or vos), and is from the lineage parameter, or is built up from the list of fileURIs accepted from fits2caom2.

Reply:
Ah - stupid question... :-). The real question is why only vos: and not also ad:?

Reply:
The method get_cadc_headers handles the distinction between ad and file. vos is not necessarily cadc, but cadc is definitely file and ad, which is how I decided to call get_vos_headers at the same level of abstraction as get_cadc_headers. Does that make sense?

Reply:
Yes, thank you.